core/src/main/java/io/grpc/internal/ServiceConfigUtil.java

```diff
@@ -313,7 +313,7 @@ static Integer getMaxResponseMessageBytesFromMethodConfig(Map<String, Object> me
           {"xds" :
```

Comment:
For the current code this needs to be "xds_experimental" right?

Reply:
This is a general form of lb config. The policy name can be arbitrary. Currently the grpc library supported policy names are: round_robin and grpclb, and we are going to support xds_experimental/xds.